core/block_rendering_rewrite/measurables.js

```diff
@@ -26,6 +26,15 @@ Blockly.BlockRendering.Measurable.prototype.isField = function() {
   return this.type == 'field';
 };
 
+/**
```

Comment:
Note for later: we *may* remove these helper functions if all of them end up being direct checks against types.

Reply:
Made a gnosis ticket in cleanup section and added todo